pkg/storage-init/storage-init.sh

```diff
@@ -6,17 +6,6 @@
 PERSISTDIR=/var/persist
 CONFIGDIR=/var/config
 
-# The only bit of initialization we do is to point containerd to /persist
```

Comment:
When we run the installer from a USB stick do we already have a /persist?

Reply:
We don't but it doesn't matter @eriknordmark -- if we don't have /persist -- then the content of /var/persist will reside on ramfs. That's more than enough for containerd.

In addition installer actually doesn't use containerd -- it simply uses oci runc.